examples/bri-3/src/bri/workgroup/workflows/agents/workflowsStorage.agent.ts

```diff
@@ -49,14 +53,14 @@ export class WorkflowStorageAgent {
     });
   }
 
-  async storeNewWorkflow(workflow: Workflow): Promise<Workflow> {
+  storeNewWorkflow(workflow: Workflow): PrismaPromise<WorkflowModel> {
```

Comment:
not sure i understand changes to this method, could you please add more details?